app/questionnaire/questionnaire_schema.py

```diff
@@ -914,6 +837,19 @@ def is_repeating_answer(
             or self.is_answer_dynamic(answer_id)
         )
 
+    def get_list_name_for_answer_id(self, answer_id: str) -> str | None:
+        """
+        Returns the name of the list if the answer is dynamic/repeating and none otherwise
+        """
+        block_id: str = self.get_block_for_answer_id(answer_id)["id"]  # type: ignore
```

Comment:
We probably need to add a short description of type ignore here and before line 850.

Reply:
Added one comment to cover both cases